platform/pulp/platform/fields.py

```diff
@@ -0,0 +1,96 @@
+"""
+Custom django fields provided by pulp platform
+"""
+import json
+
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.translation import six, ugettext_lazy as _
+
+
+def _parse_json(json_string):
```

Comment:
Instead of a separate function, perhaps you could put this code in `JSONField.to_python()` and call `self.to_python()` from `JSONField.from_db_value()`.